### PR TITLE
RUMM-464 update documentation

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: docs/rum_collection.md [hunk 1]
```diff
@@ -95,26 +95,58 @@ Depending on your application's architecture, you can choose one of several impl
...
+        try {
+          // do load the resource
+          GlobalRum.get().stopResource(resourceKey, resourceKind, additionalAttributes)
+        } catch (e : Exception) {
+          GlobalRum.get().stopResourceWithError(resourceKey, message, origin, e)
+        }
+      }
+   ```
+  
+  To track user actions, call the `RumMonitor#addUserAction`, or for continuous actions, call the `RumMonitor#startUserAction` and `RumMonitor#stopUserAction`, as follow:
```
Comment (mariusc83) [inline docs/rum_collection.md:135]: ... as follows:

File: docs/rum_collection.md [hunk 2]
```diff
@@ -95,26 +95,58 @@ Depending on your application's architecture, you can choose one of several impl
...
         GlobalRum.get().startView(viewKey, viewName, viewAttributes)        
       }
-      ...
+      
       fun onPause(){
         GlobalRum.get().stopView(viewKey, viewAttributes)        
       }
    ```
+  
+  To track resources, call the `RumMonitor#startResource` when the resource starts being loaded, and `RumMonitor#stopResource` when it is fully loaded, or `RumMonitor#stopResourceWithError` if an error occurs while loading the resource, as follow:
```
Comment (mariusc83) [inline docs/rum_collection.md:121]: ... as follows:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,6 +5,58 @@ First of all, thanks for contributing!
...
+This project hosts the following modules:
+
+  - `dd-sdk-android`: the main library implementing all Datadog features (Logs, Traces, RUM, Crash reports);
+  - `dd-sdk-android-timber`: a lightweight library providing a bridge integration between `dd-sdk-android` and [Timber](https://github.com/JakeWharton/timber);
+  - `dd-sdk-android-ktx`: a set of Kotlin extensions to make the `dd-sdk-android` library more Kotlin friendly;
+  - `instrumented/benchmark`: a test module to verify the performance of the library;
+  - `instrumented/integration`: a test module with integration tests using Espresso;
+  - `tools/detekt`: a few custom [Detekt](https://github.com/arturbosch/detekt) static analysis rules;
+  - `tools/noopfactory`: an annotation processor generating no-op implementation of interfaces;
+  - `tools/unit`: a utility library with code to help writing unit tests;
```
Comment (mariusc83) [inline CONTRIBUTING.md:25]:
```suggestion
  - `tools/unit`: an utility library with code to help writing unit tests;
```

Comment (xgouchet) [reply CONTRIBUTING.md:25]: Nope, in english, it's "a utility" ;)
cf https://www.a-or-an.com/a_an/utility

**Pull request merged**